airflow-core/src/airflow/api_fastapi/core_api/openapi/v2-rest-api-generated.yaml

```diff
@@ -12688,7 +12722,7 @@ components:
           - $ref: '#/components/schemas/TimeDelta'
           - type: 'null'
         retry_exponential_backoff:
-          type: boolean
+          type: number
```

Comment:
Can you pleae add a newsfragment about this. While I understand it should **work** without breaking installations that have "true" / "false" set, I think it's worth to add information about this to the newsfragment for Airlfow 3.2.